packages/tfjs-node-helpers/src/testing/metrics/roc-auc.ts

```diff
@@ -0,0 +1,40 @@
+import { binarize } from '../../utils/binarize';
+import { Metric } from '../metric';
+import { MetricCalculator, MetricCalculatorData } from '../metric-calculator';
+import { FPRMetricCalculator } from './fpr';
+import { RecallMetricCalculator } from './recall';
+
+export class ROCAUCMetricCalculator extends MetricCalculator {
+  constructor(private thresholdNumber: number = 21) {
```

Comment:
@VladimirGlinskii Let's rename this to `numberOfThresholds`. Also, why is it `21` by default?

Reply:
Same as above